scripts/functional-tests.sh

```diff
@@ -344,31 +344,6 @@ test_model_print(){
         }
     '
 
-    # validate that we print the model from the CLI
```

Comment:
Why is this being removed?

Reply:
Agree

Reply:
this is testing whether the user can put something like baz.gguf, this should not work and will not. The new code only allows  -m which is in the server

Reply:
`ilab chat -m bar` should really never have worked, this PR makes it so that -m only works for real, fully qualified models.

Reply:
sure